core/pkg/eval/json_evaluator.go

```diff
@@ -49,7 +49,17 @@ const (
 	Disabled = "DISABLED"
 )
 
-func NewJSONEvaluator(logger *logger.Logger, s *store.Flags) *JSONEvaluator {
+type JSONEvaluatorOption func(je *JSONEvaluator)
+
+func WithEvaluator(aliases []string, evalFunc func(interface{}, interface{}) interface{}) JSONEvaluatorOption {
```

Comment:
This is a slight behavioral change since previously we only had a single alias for an operator.
```suggestion
func WithEvaluator(aliases string, evalFunc func(interface{}, interface{}) interface{}) JSONEvaluatorOption {
```

Reply:
the possibility for providing a list of aliases was done as preparation for introducing an alias for the `fractionalEvaluation` (see https://github.com/open-feature/flagd/pull/658#discussion_r1196717542), but we can also introduce this later when we work on that specific change

Reply:
as discussed, changed it to `string` instead of `[]string`

Reply:
Should I create a follow-up for adding an alias? It would be worth it since `fractionalEvaluation` has a different casing than the others. It looks a bit out of place in the docs I was working on.

https://github.com/open-feature/flagd/pull/681